pact/http/provider/build.gradle.kts

```diff
@@ -17,3 +18,12 @@ dependencies {
     testImplementation("au.com.dius.pact.provider:junit5")
     testImplementation("io.mockk:mockk")
 }
+
+project.version = "3.2.1"
+
+tasks {
+    withType<Test> {
+        systemProperty("pact.verifier.publishResults", "true")
+        systemProperty("pact.provider.version", project.version)
```

Comment:
Same questions as for the consumers: is `project.version` needed, or could it be set directly without issues?

A bit of my thoughts on why I would prefer it without the `project.version`, if possible: 
For a real microservice I would not set a project version. If a version is needed (e.g. if the Service JAR needs to be published before release), I usually just set it during release builds. Not sure how common this approach is though.

Reply:
I've inlined the versions now.